keps/sig-api-machinery/1929-built-in-default/README.md

```diff
@@ -400,10 +402,12 @@ which would have the following behavior, both for CRD types and built-in types:
 
 #### Non-pointer fields
 
-Non-pointer fields are also important, since they have a specific unmarshalling mechanism in Go.
+Non-pointer fields are also important, since they have a specific
```

Comment:
what does "are also important" mean here? You mean that they need special semantics (= "always there") and therefore we have to come with something equivalent for CRDs.